react-ui-kit/src/Inputs/BaseField/Input.tsx

```diff
@@ -29,9 +17,10 @@ interface IInputProps extends IInputDefaultProps {
   iconIsActive?: boolean;
   setIconIsActive?: Dispatch<SetStateAction<boolean>>;
   number?: boolean;
-  inputDate?: boolean;
   range?: boolean;
   activeInformer?: boolean;
+  value: any;
+  errorActive?: boolean;
 }
 
 export const Input = ({
```

Comment:
Надо изменить тип. Посмотри, например, компонент Stepper
```suggestion
export const Input: FC<IInputProps> = ({
```
JSX.Element не надо указывать, это всё уже есть в FC

От enum'ов тоже надо уйти